instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Login = (props) => {
+  return (
+    <div className="page login">
+      <input className="username-input" defaultText="username" />
```

Comment:
We can use `placeholder="username"` instead of `defaultText` to show "username" in the input as a placeholder.